replicatedhq/ship
# remove ship init helm ref support

What I Did
------------
Removed support for `ship init` with a helm ref. This removes commands like `ship init stable/mysql --chart-version=0.8.1`. In doing so, this resolves #500.

How I Did it
------------
I removed the helm fetch code from `pkg/specs/apptype/determine_type.go` and the code to save/load helm chart version/chart urls from `pkg/state/*`. `pkg/cli/init.go` was edited to no longer accept those flags and to have a message that no longer mentions `helm fetch` style refs.

How to verify it
------------
Attempt to run `ship init stable/mysql` and observe error.

Description for the Changelog
------------
Removed support for `ship init` with a helm ref


Picture of a Boat (not required but encouraged)
------------
![image](https://user-images.githubusercontent.com/2318911/44744652-7649ff00-aaba-11e8-829e-c374224a331f.png)
